sympy/physics/mechanics/rigidbody.py

```diff
@@ -182,7 +182,7 @@ def angular_momentum(self, point, frame):
         The angular momentum H of a rigid body B about some point O in a frame
         N is given by:
 
-            H = I . w + r x Mv
+            `H = dot(I, w) + cross(r, M) * v`
```

Comment:
Should it not be `cross(r, Mv)`? I think cross products are only defined on vectors.

Reply:
Sorry by mistake, I corrected it